pkg/controller/virtinformers.go

```diff
@@ -543,10 +543,43 @@ func (f *kubeInformerFactory) VirtualMachine() cache.SharedIndexInformer {
 	})
 }
 
+func GetVirtualMachineExportInformerIndexers() cache.Indexers {
+	return cache.Indexers{
+		"pvc": func(obj interface{}) ([]string, error) {
+			export, ok := obj.(*exportv1.VirtualMachineExport)
+			if !ok {
+				return nil, unexpectedObjectError
+			}
+
+			if export.Spec.Source.APIGroup == nil ||
+				*export.Spec.Source.APIGroup == "" || *export.Spec.Source.APIGroup == "v1" &&
+				export.Spec.Source.Kind == "PersistentVolumeClaim" {
+				return []string{fmt.Sprintf("%s/%s", export.Namespace, export.Spec.Source.Name)}, nil
+			}
+
+			return nil, nil
+		},
+		"vmsnapshot": func(obj interface{}) ([]string, error) {
```

Comment:
I don't see where this is referenced

Reply:
Wow, that is terrible, I completely didn't add the handler for that....